### DISCUSSION TITLE
ikvm.org

### DISCUSSION THREAD

**Discussion by wasabii** [ikvmnet/ikvm#421] [Announcements]
Had purchased this domain awhile ago. Just threw a site up on it. Using Nuxt and Docus. Idea stolen from MassTransit.

Since the ikvmnet organization has multiple projects, it makes sense to put all of the documentation on the same site. So, there's a ikvmnet.github.io repository with the Nuxt install. Single branch. Deploys to Github Pages. Static generation.

In CI/CD, it pulls the 'doc.zip' release artifact from the latest ikvm and ikvm-maven repos and extracts them into content/32.ikvm and content/33.ikvm respectively. This artifacts are published from each project's release, from the doc/ directory. This lets us throw MD files into the doc/ folder in the root of each repo, and hae those docs merged into the Nuxt site.

Will have the ikvm and ikvm-maven release trigger the action in the docs site.

So: ikvm repository now has a doc/ folder, and we can begin some basic markdown documentation in it. It would be good to at least get the basic instructions up.

@NightOwl888 @AliveDevil

**Comment by AliveDevil** [comment]
Nice. Will take a look to improve that once I have more time.
